### PR TITLE
+vim.org

### LINKED ISSUE #188: +vim.org
related to Top 300

### PR DESCRIPTION
**Pull request by getneil**
resolves #188

### CODE REVIEW

**Comment by getneil** [comment]
the error doesnt seem helpful atm, but this builds and test correctly in my local.

Screenshot 2023-02-10 at 4 17 22 PM

issue seems to be useGithubApi related, graphql query. ...investigating

**Comment by jhheider** [comment]
Given that there's 15,000 tags, and the API returns them oldest first, it might just be too many. I can't easily see what the oldest tags look like, but it might be that the API limits out before it finds one it can read. This *looks* like it should scan them all, though: https://github.com/teaxyz/cli/blob/12f482cf8ffaa99f646317513ca1230e113f9391/src/hooks/useGitHubAPI.ts#L50-L91

**Comment by getneil** [comment]
I see. It might be better to manually specify versions right?

File: projects/vim.org/package.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      --enable-luainterp
+    make
+    make install prefix={{prefix}}
+    if test "{{hw.platform}}+{{hw.arch}}" = "linux+x86-64"; then
+      cd {{prefix}}/bin
+      ln -s vim vi
+    fi
+test:
+  script: |
+    vim --version
```
Comment (getneil) [inline projects/vim.org/package.yml]: original intended test:
```
test:
  script: |
    vim --version
    vi --version
    echo "hello world" >> ./test.txt
    echo ":s/hello/test/" >> ./commands.vim
    echo ":wq" >> ./commands.vim
    vim -T dumb -S ./commands.vim ./test.txt
    test "$(cat ./test.txt)" = "test world"
```

still figuring out but it freezes on `vim -T dum ...`

Comment (mxcl) [reply projects/vim.org/package.yml]: probs waiting on stdin input even though I assume that's what `-T dumb` is meant to prevent…?

Comment (jhheider) [reply projects/vim.org/package.yml]: I suspect `-T dumb` is the terminal to use; I found great difficulty testing ncurses applications even with the terminal dummied out.

**Comment by jhheider** [comment]
Beautiful teamwork, everyone!

**Pull request merged**